homeassistant/components/xiaomi_aqara.py

```diff
@@ -189,23 +195,33 @@ def remove_device_service(call):
 class XiaomiDevice(Entity):
     """Representation a base Xiaomi device."""
 
-    def __init__(self, device, name, xiaomi_hub):
+    def __init__(self, device, name, hass, xiaomi_hub):
         """Initialize the xiaomi device."""
         self._state = None
+        self._lock = threading.Lock()
+        self._hass = hass
```

Comment:
There is no need to pass `hass` down. The device will get `hass` added on the entity when it is added to Home Assistant.

Reply:
Ok, removed. Using self.hass instead after added_to_hass() hook.